SU-SWS/stanford_media
# D8CORE-638 D8CORE-801 Tweaks to WYSIWYG media

# READY FOR REVIEW

# Summary
- Changed the media library icon in ckeditor toolbar
- Modified styles for the "Edit Media" button in the wysiwyg.

# Need Review By (Date)
- 11/25

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear cache
1. in a wysiwyg check that the icon for media library is only an image
1. add an image in the wysiwyg. Verify the button is centered on the image and is more obvious to click.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
